.gitlab-ci.yml

```diff
@@ -18,10 +18,9 @@ variables:
 
 stages:
   - test
-  - deploy_dev
+  - deploy
   - integration_test
-  - release_integration
-  - release_staging
+  - release
```

Comment:
nit: order these chronologically

Reply:
In my mind they are: `test` first, if pass `deploy`, if deployed `integration_test`

Reply:
Ah, good point. OK then, go ahead.